tokio/src/runtime/task/state.rs

```diff
@@ -54,11 +54,11 @@ const REF_ONE: usize = 1 << REF_COUNT_SHIFT;
 
 /// State a task is initialized with
 ///
-/// A task is initialized with two references: one for the scheduler and one for
+/// A task is initialized with three references: two for the scheduler and one for
```

Comment:
We can probably be more explicit here. "One for `OwnedTasks` (held by the scheduler), one for `Notified<_>` (used to poll the task), and one for `JoinHandle`.

or something.